src/components/Footer.js

```diff
@@ -61,16 +97,15 @@ const Footer = () => {
                 </IconButton>
                 <IconButton color="white">
                   <InstagramIcon />
-                </IconButton> 
+                </IconButton>
                 <IconButton color="white">
                   <FacebookIcon />
                 </IconButton>
               </Box>
               <Typography variant="caption" color="white" textAlign={"center"}>
-                Copyright ©{new Date().getFullYear()}.  NOMADS.SOLUTIONS
+                Copyright ©{new Date().getFullYear()}. NOMADS.SOLUTIONS
```

Comment:
same issue

Reply:
For now i am approving this PR.
> Suggestion 
-  check the changes you have made are correct or not before creating any PR. Do not make any unnecessary changes in codebase apart from the feature/bug you are working on. @Sky-De 
- Apart from this you PR  looks good

## Thank you for contributing :heartpulse:  , . If you need any help go to discussion Tab and put your question there.

Reply:
@techy1999 thanks, but all those changes happened because of prettier, and i mentioned that, also according to issue #88 you will need all those changes on all code source later,
But for now it will be ok, don't worry it